src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.HashMap;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(HashMap storage, FruitTransaction transaction) {
```

Comment:
The method signature `handle(HashMap storage, FruitTransaction transaction)` uses a raw type for the `HashMap`. Use generics to specify the type of keys and values in the `HashMap` to ensure type safety.